<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NmList</class>
 <widget class="QDialog" name="NmList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QLabel" name="labelAll">
     <property name="text">
      <string>All information</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listActive"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelActive">
     <property name="text">
      <string>Active connections</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListView" name="listWifi"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelWifi">
     <property name="text">
      <string>Available wireless</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTreeView" name="treeView">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
